infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -129,10 +131,37 @@ public LogUnitServer(ServerContext serverContext) {
     /**
      * Service an incoming request for maximum global address the log unit server has written.
      */
-    @ServerHandler(type = CorfuMsgType.TAIL_REQUEST)
-    public void handleTailRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
+    @ServerHandler(type = CorfuMsgType.LOG_TAIL_REQUEST)
+    public void handleLogTailRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
```

Comment:
please rename `r` to `router`

Reply:
If you see all handlers are using this same naming, I think it is more consistent to leave it like it is in all places, this was not added by me.